bigbluebutton-html5/imports/ui/components/audio/local-echo/service.js

```diff
@@ -27,8 +27,14 @@ const playEchoStream = async (stream, loopbackAgent = null) => {
         loopbackAgent.stop();
       }
     }
-
-    audioElement.srcObject = streamToPlay;
+    const audioCtx = new AudioContext();
```

Comment:
I'm not too sure the new items (context, source and node) will be properly cleaned up/released this way. They might dangle/leak without explicit `close` calls.